### PR TITLE
Update move-tables.md

### PR DESCRIPTION
**Pull request by zhe-wish**
To insert data we should use mysql < ../common/insert_commerce_data.sql
Otherwise the tables would be empty.

Reference: https://github.com/vitessio/vitess/blob/master/examples/local/README.md

### CODE REVIEW

**Comment by hallaroo** [comment]
Hi @zhe-wish to get this PR reviewed and merged there is one thing to do first. The DCO check needs to pass.

To do this you'll need to follow the steps here: https://github.com/vitessio/website/pull/749/checks?check_run_id=2435303496

Once that check is passed we'll be able to review the requested change and then get it merged. Please reach out if you have any questions or issues with this. Thanks!

**Comment by askdba** [comment]
@zhe-wish could you please fix the DCO which is required by the license.

File: content/en/docs/user-guides/migration/move-tables.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Let's start by simulating this situation by loading sample data:
 
 ```sh
 # On helm, local and operator installs: 
-mysql --table < ../common/insert_commerce_data.sql
+mysql < ../common/insert_commerce_data.sql
```
Comment (deepthi) [inline content/en/docs/user-guides/migration/move-tables.md:21]: What OS are you on? On Ubuntu both the before and after versions of this command work fine.

**Comment by deepthi** [comment]
Closing as stale PR. You may re-open and respond to the feedback if you would like.

**Pull request closed without merging**